tests/test_igus_dryve.py

```diff
@@ -1,37 +1,44 @@
-import logging
+import types
 import unittest
 
-from lsst.ts.linearstage.controllers.igus_dryve import IgusLinearStageStepper
-from lsst.ts.linearstage.controllers.igus_dryve.telegrams import (  # telegrams_read_errs,
+from lsst.ts.linearstage.controllers.igus_dryve import Igus

```

Comment:
I suggest omitting `igus_dryve` from this import, since the module name is an internal detail. In fact that applies to all of these imports. This should work fine, since you moved all those modules up out of the igus directory:
```
from lsst.ts.linearstage.controllers import Igus, telegrams_read, telegrams_write, interpret_read_telegram
```